README.md

```diff
@@ -6,12 +6,118 @@ output:
+* Application can make use of JS power by customizing 'www/js/custom.js'
+
+Create and run sample app for full features demo
+
+```{r}
+periscope2::create_application(name = "demo_app", location = ".", sample_app = T, right_sidebar = T)
+shiny::runApp("./demo_app/", launch.browser = T)
+```
+
+### periscope vs periscope2
+[periscope](https://cran.r-project.org/web/packages/periscope/index.html) is widely regarded and used an enterprise application development aid framework for providing the user with the ability to build a robust shiny applications and dashboards.
+
+[periscope](https://cran.r-project.org/web/packages/periscope/index.html) is the direct ancestor of periscope2 in which periscope2 took most of its basic philosophy and generated applications structure.
+
+However; as periscope2 is the next generation (iteration) of [periscope](https://cran.r-project.org/web/packages/periscope/index.html) package, they differ in many aspects as follow:
+
+#### Underlying frameworks
+[periscope](https://cran.r-project.org/web/packages/periscope/index.html) is depending on bootstrap 3 supported shiny packages:
+
+- [shinydashboard](https://cran.r-project.org/web/packages/shinydashboard/index.html)
+- [shinydashboard Plus](https://cran.r-project.org/web/packages/shinydashboardPlus/index.html)
+- [shinyBS](https://cran.r-project.org/web/packages/shinyBS/index.html)
+
+While periscope2 is depending only on bootstrap 4 supported package [bs4Dash](https://cran.r-project.org/web/packages/bs4Dash/index.html) to cover the same functions that are presented by [periscope](https://cran.r-project.org/web/packages/periscope/index.html) dependencies and more (i.e dashboard functionality, alerts, tool-tips, etc..) in more modernized look and feel
+
+----
+
+#### Generated applications layout
+[periscope](https://cran.r-project.org/web/packages/periscope/index.html) full generated app layout is simple as follow:
+
+- Left sidebar
+- Right sidebar
+- Header
+- Body
+
+All above parts are customizable except for header, user can only update title part in it
+
+However, periscope2 full generated app layout has the following structure:
+
+- Left sidebar
+- Right sidebar
+- Header
+- Footer
+- Body
+
+For periscope2, each part has it own UI R file that can customized and updated alone. Aslo, each part has its own dedicated alert panel except for header that has an additional alert panel for announcements framework.
+
+User can adapt layout for both packages generated apps easily via related functions (refer to `r ?periscope` and `r ?periscope2` for more info)
+
+-----
+
+#### Generated applications code structure
+
+- Regardless files contents, both packages generated applications share the same first level folder structure:
+
+<center><img src="./vignettes/figures/App_Structure.png"></center>
+
+- After that, there are differences in next level structure as follow:
```

Comment:
change to "Differences in the next level of structure are:"

Reply:
done